tutorials/horizontally-scale-mysql-database-backend-with-google-cloud-sql-and-proxysql/index.md

```diff
@@ -0,0 +1,325 @@
+	UPDATE mysql_servers SET weight = 20 WHERE hostname = '[IP_ADDRESS]';
+	LOAD MYSQL SERVERS TO RUNTIME;
+	SAVE MYSQL SERVERS TO DISK;
+	```
+
+	Replace `[IP_ADDRESS_1]`, `[IP_ADDRESS_2]`, `[IP_ADDRESS]` with the ip addresses of read replica #1, read replica #2, and the master instance respectively. The weights are calculated as follows:
+
+	```
+	P(Server # Receives the Query) = (Weight of Server #) / (Total Weight of the Host Group)
+	```
+
+	Read replica #1 has a weight of 40, so its chance of receiving a query is 40/100 = 0.4 (40%). Similarly, read replica #2 has a chance of 40% and the master instance 20%.
+
+### Scaling Dynamically
+
+You can add or remove read replicas any time you want. When you are adding a read replica, create it in Cloud SQL as instructed above, add it to the `mysql_server` table, and update the rules (and/or weights) to route traffic to it. Removing a read replica, however, takes a few additional steps:
+
+1. Update the status of the read replica to `OFFLINE_SOFT`:
+
+	```
+	UPDATE mysql_servers SET status = 'OFFLINE_SOFT' WHERE hostname = '[IP_ADDRESS]';
+	LOAD MYSQL SERVERS TO RUNTIME;
+	SAVE MYSQL SERVERS TO DISK;
+	```
+
+	`OFFLINE_SOFT` status flag instructs ProxySQL to stop sending new queries to the read replica. Existing connections still persist. Monitor the status of the read replica in Cloud Console (Go to the [Cloud SQL Instances page](https://console.cloud.google.com/sql/instances), click the read replica you would like to remove, and the performance chart is available in the Instance Details page). You should see that your read replica is no longer used by your application. Note that setting the read replica to offline in ProxySQL does not stop it from running in Cloud SQL; you are still being charged for the replica. See steps below to remove it from Cloud SQL.
+
+2. If you do not plan to use the read replica any more in the future, delete it from `mysql_servers` table:
+
+	```
+	DELETE FROM mysql_servers WHERE hostname = '[IP_ADDRESS]';
+	LOAD MYSQL SERVERS TO RUNTIME;
+	SAVE MYSQL SERVERS TO DISK;
+	```
+
+	All the traffic will be terminated immediately, so you should proceed with deletion only after you are certain that the read replica has completed all the queries sent its way. Use the performance chart (`MySQL queries`, `active connections`, etc.) to monitor the status.
+
+	Then remove it from Cloud SQL:
+
+		a. Go to the [Cloud SQL Instances page](https://console.cloud.google.com/sql/instances) and click the read replica you would like to remove.
+		b. Click **Delete**. Follow the instructions on screen to delete the instance.
+
+3. Update the rules and weights accordingly.
+
+Unlike the master instance, read replicas cannot be stopped (and then restarted later) in Cloud SQL. If you plan to add a read replica after deletion, create a new one and update ProxySQL settings. Master instance cannot be stopped or deleted until all the read replicas are deleted.
```

Comment:
Add *The*:
* "...create a new one and update the ProxySQL settings."
* "The master instance cannot be stopped or deleted..."